packages/utils/src/hooks.js

```diff
@@ -1,4 +1,5 @@
 import { useRef, useEffect, useState } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This will likely go away once `@edx/frontend-platform` is added back as a peer dependency.